ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installer-rehearse-4.21.yaml

```diff
@@ -139,6 +139,17 @@ tests:
     env:
       EXTRACT_MANIFEST_INCLUDED: "true"
     workflow: cucushift-installer-rehearse-azure-stack-upi
+- as: installer-rehearse-aws-gp3-throughput
+  cron: 0 2 * * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: cucushift-installer-check-aws-gp3-throughput
+    workflow: cucushift-installer-rehearse-aws-ipi
```

Comment:
The throughput parameter should be configured and injected into the installation step.

In addition, an `AWS_GP3_XXX` parameter should be added to `ipi-conf-aws-ref.yaml` and used by the `ipi-conf-aws-commands.sh`.